framework/core/js/dist-typings/admin/components/HeaderPrimary.d.ts

```diff
@@ -4,6 +4,7 @@
  */
 export default class HeaderPrimary extends Component<import("../../common/Component").ComponentAttrs, undefined> {
     constructor();
+    view(): JSX.Element;
```

Comment:
All of these should really be `Mithril.Children` instead (return type manually defined in the method declaration).